packages/conf-rust-2021/conf-rust-2021.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+maintainer: "contact@trili.tech"
+authors: [
+  "TriliTech"
+  "Simon Cruanes"
+]
+homepage: "https://github.com/ocaml/opam-repository"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "MIT"
+extra-files: ["test.rs" "sha256=536e506bb90914c243a12b397b9a998f85ae2cbd9ba02dfd03a9e155ca5ca0f4"]
+build: [
+  ["cargo" "--version"]
+  ["rustc" "--edition" "2021" "test.rs"]
+]
+depexts: [
+  ["cargo"] {os-distribution = "centos" & os-version >= "8"}
+  ["cargo"] {os-distribution = "ol" & os-version >= "8"}
+  ["cargo"] {os-distribution = "fedora"}
+  ["cargo"] {os-family = "suse"}
+  ["cargo"] {os-family = "debian"}
```

Comment:
This is not available on all debians. I think this line should be

```suggestion
  ["cargo"] {os-family = "debian" & os-version > "10"}
```

Reply:
I am not sure that'll do much good though. The default Rust version that ships with Debian 11 (bullseye) is also too old (1.48). However, if one were to amend the constraint to `>= 12`, that means no `cargo` depext is suggested at all.

Reply:
To put into perspective how the error looks: https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/edb4003792ecd6df75f29618ab0d3d63aaac66d1/variant/distributions,debian-10,conf-rust-2021.1